<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWidget</class>
 <widget class="QWidget" name="MainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>96</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout" columnstretch="0,0,0">
   <item row="0" column="0">
    <widget class="QLabel" name="srcLbl">
     <property name="font">
      <font>
       <family>Consolas</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>SRC:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="dstLbl">
     <property name="font">
      <font>
       <family>Consolas</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>DST:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="MyLineEditDir" name="dstDir">
     <property name="font">
      <font>
       <family>Consolas</family>
       <pointsize>11</pointsize>
      </font>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="MyLineEditDir" name="srcDir">
     <property name="font">
      <font>
       <family>Consolas</family>
       <pointsize>11</pointsize>
      </font>
     </property>
    </widget>
   </item>
   <item row="0" column="2" rowspan="2">
    <widget class="QPushButton" name="encodeConv">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Conv</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QLabel" name="stateLbl">
     <property name="font">
      <font>
       <family>Consolas</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyLineEditDir</class>
   <extends>QLineEdit</extends>
   <header>mylineeditdir.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>srcDir</tabstop>
  <tabstop>dstDir</tabstop>
  <tabstop>encodeConv</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
